public/js/eventsPage.js

```diff
@@ -29,7 +29,14 @@ function getEvents() {
  */
 function displayEventsHandler(events) {
     let eventInformation = ''
+    let noEvents = document.querySelector('.no-events')
```

Comment:
could go straight into the evets div rather than having another span. Just a bit cleaner.